libraries/redcore/layouts/searchtools/default.php

```diff
@@ -23,7 +23,7 @@
 	'orderFieldSelector'  => '#list_fullordering'
 );
 
-$data['options'] = array_unique(array_merge($customOptions, $data['options']));
+$data['options'] = array_replace($customOptions, $data['options']);
```

Comment:
can you please update branch (or add me to your fork contributor list so I can update your base when needed)